dspace/config/modules/usage-statistics.cfg

```diff
@@ -1,13 +1,14 @@
 #---------------------------------------------------------------#
 #--------------USAGE STATISTICS CONFIGURATIONS------------------#
 #---------------------------------------------------------------#
-# These configs are only used by the DSpace interfaces which    #
-# track usage statistics (Solr)                                 #
+# These properties are only used by the DSpace interfaces which #
+# track usage statistics (Solr).                                #
 # See also: solr-statistics.cfg                                 #
 #---------------------------------------------------------------#
 
-# The location for the Geo Database retrieved on update/installation
-usage-statistics.dbfile = ${dspace.dir}/config/GeoLite2-City.mmdb
+# The location for the database used to map client addresses to approximate
+# physical locations.
+usage-statistics.dbfile = /usr/share/GeoIP/GeoLite2-City.mmdb
```

Comment:
Just a quick note...is this a "standard" path where GeoLite2 tends to install itself?  If so, this is OK to preconfigure (though we may want to enhance the comments here to link back to GeoLite2 install docs, if plausible).  If there is no "standard" location for GeoLite2, I wonder if we should consider commenting this out by default (if that doesn't break things), to essentially force people to set this to an appropriate setting after they install GeoLite2.

I guess I'm just trying to think about the most appropriate error message for folks to see here if things end up misconfigured.  Is it better to see a "File not found" (pointing at this path) error, or is there somewhere we can simply check if `usage-statistics.dbfile` is set...and if not, throw an error saying that it is "required for Solr Statistics" and that GeoLite2 must be installed.

Reply:
That is where MaxMind's updater places the databases on my development system, and it looks like a standard sort of place which I would expect to be used on any distro.  Of course it won't work on Windows.  So it might be best to comment out as you suggest, and make certain that DSpace gracefully recovers or gives a useful message if it is not configured.

The documentation does need updating, and I'll do that if/when this patch is merged.

Reply:
SolrLoggerServiceImpl logs a complaint and skips the calls to GeoIP if the database location is unconfigured.